src/bootstrap/config.rs

```diff
@@ -84,7 +95,10 @@ pub struct Config {
     pub jobs: Option<u32>,
     pub cmd: Subcommand,
     pub incremental: bool,
-    pub dry_run: bool,
+    // #[cfg(not(test))]
+    // dry_run: DryRun,
+    // #[cfg(test)]
```

Comment:
Stray comments?

Reply:
Yes, thanks :) I made this private so the compiler would suggest calling the new `dry_run()` function and I could use `cargo fix` to apply the suggestions.